baseMdl/mdl.py

```diff
@@ -1,62 +1,7 @@
-import random
 import numpy as np
-import pygame
 
```

Comment:
Aquí agregar `import os` para que jale lo de guardar los modelos en el directorio de "savedModels"
```suggestion
import os

```